### ISSUE TITLE
GeoSpatial attributes Visible when Viewing other Resource Types

### ISSUE THREAD

**Issue by annikaLiving** [bcgov/ckan-ui#312]
The following fields are visible in View mode for Tabular, Webservices/API and Application Records, yet only belong to Resources for BCGW Geospatial datasets.

These fields should not be visible if not populated as many Geospatial datasets also do not include these.

* Object Short Name
* Object Table Comments
* Details

**Comment by SteveChapmanBCDX** [comment]
We should probably do an audit to make sure that each resource type is displaying relevant fields (and no other fields).

**Comment by annikaLiving** [comment]
@SteveChapmanBCDX good point as it see Application, Webservices and Tabular are all showing the same fields and this is not the case in Cat Classic. Where as Application should only have a few fields.

* Did the UI mock ups I did ~6 months ago get written up as these outlined what fields should be for each resource type?

**Comment by SteveChapmanBCDX** [comment]
@annikaLiving - We'll have to confirm with @yuisotozaki.  I'm pretty sure that he did separate mockups for each of the resource types and then modified the views based on the review meetings that we did last fall (Yui, you, me, Brandon, DAs).

**Comment by yuisotozaki** [comment]
Yes the separate fields per resource type was discussed amongst the team before.
No, we didn't create a design for each resource type.

**Comment by annikaLiving** [comment]
@TerryLanktree this is one of the tix we are going to discuss this afternoon.

**Comment by annikaLiving** [comment]
These fields are wrong on bcdc_types where not geographic and bcgw and thus should be removed.

**Comment by sheilamv** [comment]
https://dpdd.atlassian.net/browse/DDS-1071
